__tests__/api/request_test.js

```diff
@@ -0,0 +1,36 @@
+let requests = require('../../api/requests');
+let moment = require('moment');
+jest.mock('googleapis');
+let google = require('googleapis');
+
+// This needs to be using a specific time 
+// Test delays means the times are sometimes out of sync
+let originalDate = moment("2013-08-26 16:55:00");
+let dateTime = originalDate.toISOString();
+let date = originalDate.format('YYYY-MM-DD');
+
+describe("Checking the get of google API information is returned in the correct format", function () {
+
+  it("Should return three responses for getting all learningEvents", function (done) {
+    requests.listEvents().then(response => {
```

Comment:
We should able at least to test that the `auth` parameter we pass to `request.listEvents` is being used in `calendar.events.list()`

Reply:
Yeah good catch, we'll do that now :)